tests_integration/test_data/projects/snowpark_external_access/app/app.py

```diff
@@ -1,5 +1,5 @@
 import _snowflake
-import requests
+from http.client import HTTPSConnection
```

Comment:
FYI `connexion` has same API and no license problems

Reply:
🆗 If I'm able to do it without external dep it is even better.